### PR TITLE
Add destination and offset methods

### LINKED ISSUE #2: Add `offset()` method
This method should offset the `FlatPoint` by x/y kilometers towards north/east.

### PR DESCRIPTION
**Pull request by mthh**
Hi,

This PR adds the `destination` method and the `offset` method on `FlatProjection` in order to resolve #2 and #4  .

### CODE REVIEW

**Comment by mthh** [comment]
Done.
Thanks to you for offering issues for newcomers! (and thank you for the time it probably involves!)

File: src/lib.rs [hunk 1]
```diff
@@ -285,6 +285,73 @@ impl<T: Float> FlatPoint<T> {
...
+    /// #
+    /// # assert_approx_eq!(dest_lon, 30.5013947, 0.00001);
+    /// # assert_approx_eq!(dest_lat, 50.5, 0.00001);
+    /// # assert_approx_eq!(distance, res_distance, 0.00001);
+    /// # }
+    /// ```
+    pub fn destination(&self, dist: T, bearing: T) -> FlatPoint<T> {
+        let a = (T::from(90.).unwrap() - bearing).to_radians();
+        self.offset(a.cos() * dist, a.sin() * dist)
+    }
```
[the remark this answers is not in the document]
Comment (mthh) [reply src/lib.rs]: Should I move these unit tests outside of documentation comment ?

[the remark this answers is not in the document]
Comment (Turbo87) [reply src/lib.rs]: > Should I move them outside of documentation comment ?

yeah, I think that would be better :)

[the remark this answers is not in the document]
Comment (mthh) [reply src/lib.rs]: I splitted these tests it in a 4 functions (one for each quadrant) and moved them in a separate mod.

**Comment by Turbo87** [comment]
Thanks again! :)

**Pull request merged**